cluster-autoscaler/cluster_autoscaler.go

```diff
@@ -410,7 +380,7 @@ func main() {
 		kube_leaderelection.RunOrDie(kube_leaderelection.LeaderElectionConfig{
 			Lock: &resourcelock.EndpointsLock{
 				EndpointsMeta: apiv1.ObjectMeta{
-					Namespace: "kube-system",
```

Comment:
We'll have to remember to update cluster autoscaler examples/manifests once we upgrade to a version that expects the namespace flag.